pkg/framework/capi-machines.go

```diff
@@ -0,0 +1,56 @@
+package framework
+
```

Comment:
and this is modified from machine.go from cluster-capi-operator ? 
Similarly for capimachineset.go  ?

Reply:
Yes. But how about moving these to a new folder? like pkg/framework/capi/

Reply:
I wonder if we need to create another capi folder. It looks clear now. If we add a new folder, several go files with the same name, and cases add label still need to import "github.com/openshift/cluster-api-actuator-pkg/pkg/framework"

Reply:
Okay, the current code is fine for me.